### DISCUSSION TITLE
Default split size on general compression settings

### DISCUSSION THREAD

**Discussion by ghost** [aonez/Keka#1223] [Ideas]
Hi,

I would like to suggest the feature of adding "Default split: ____ " at the general settings for the compression tab and for that configuration to stay saved until manually changed by the user, regardless of the drop-down available options.

![image](https://user-images.githubusercontent.com/94919997/220613150-d987b5d1-4080-415d-8422-48aed05960ee.png)

Right now, if I want to split a file, I have to open Keka manually, write the Split size and then drag the file to the window.

![image](https://user-images.githubusercontent.com/94919997/220613794-747037c3-a779-41f4-aadb-d138cdc37769.png)

I think it would be more convenient to have the ability to save it as a default setting.

Thank you for developing Keka, it's awesome! :)

**Comment by aonez** [comment]
Thanks for the feedback @kenzogrosvald!

That setting will mean that Keka will always split all compressions, independently of how you call it. I'm not sure this will be a good idea.

Maybe another option is to add a "Remember the last split value" option, so it only will split if you open Keka before the compression, but it will automatically be set to the last used value.
